clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/GCEUtil.groovy

```diff
@@ -1062,8 +1062,7 @@ class GCEUtil {
     def balancingMode = policy.balancingMode
     return new Backend(
       balancingMode: balancingMode,
-      maxRatePerInstance: balancingMode == GoogleLoadBalancingPolicy.BalancingMode.RATE ?
-        policy.maxRatePerInstance : null,
+      maxRatePerInstance: policy.maxRatePerInstance,
```

Comment:
`maxRatePerInstance` doesn't make sense for the `CONNECTION` [balancing mode](https://github.com/spinnaker/clouddriver/blob/master/clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/model/loadbalancing/GoogleLoadBalancingPolicy.groovy#L24), so I think we need a check for `RATE` or `UTILIZATION` here.